server/test/index.test.js

```diff
@@ -581,5 +571,69 @@ describe('Server Tests', () => {
         .expect('Content-Type', /json/);
       return expect(expected).toEqual(res.body);
     });
+    test('/api/v1/patients/:patientId route should return message Updated successfully', async () => {
+      const message = 'Updated successfully';
+      const res = await request(app)
+        .patch('/api/v1/patients/9')
+        .send({
+          firstName: 'alaa',
+          lastName: 'alser',
+          phone: '0592623088',
+          email: 'alasa@lhaser.com',
+          birthday: '1994-09-02',
+          diseases: 'no diseases',
+        })
+        .expect(200)
+        .expect('Content-Type', /json/);
+      return expect(message).toEqual(res.body.message);
+    });
+    test('/api/v1/patients/:patientId route should return error phone number is exist', async () => {
+      const message = 'phone number is exist';
```

Comment:
Don't forget to edit this test inorder to make your tests passing
```suggestion
    test('/api/v1/patients/:patientId route should return error Key (phone)=(0599010102) already exists.', async () => {
      const message = 'Key (phone)=(0599010102) already exists.';
```